### PR TITLE
Add log subscription lambda

### PR DESCRIPTION
**Pull request by npeternel**
Adds a Lambda shortcut for https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-subscriptionfilter.html

### CODE REVIEW

File: lib/shortcuts/index.js [hunk 1]
```diff
@@ -11,5 +11,6 @@ module.exports = {
   ServiceRole: require('./service-role'),
   Queue: require('./queue'),
   S3KinesisFirehose: require('./s3-kinesis-firehose'),
-  hookshot: require('./hookshot')
+  hookshot: require('./hookshot'),
+  SubscriptionFilterLambda: require('./subscription-filter-lambda')
```
Comment (rclark) [inline lib/shortcuts/index.js]: I think that `LogSubscriptionLambda` might be more informative about what this shortcut exists to implement. Thoughts?

Comment (npeternel) [reply lib/shortcuts/index.js]: totally agree with you there!

Comment (npeternel) [reply lib/shortcuts/index.js]: rolls off the tongue more easily too

**Comment by npeternel** [comment]
@rclark thanks for taking a look at this so fast.
i can't seem to generate the actual Lambda in the cloudformation fixtures. is this something i do manually or am i missing something there?

**Comment by vsmart** [comment]
@npeternel I think you run the tests like `UPDATE=true npm test`, see https://github.com/mapbox/cloudfriend/blob/37bb0a57570a5ed4449f0ba8b46472ae896282cf/test/shortcuts.test.js#L9. We should get that added to the contributing docs.

**Comment by npeternel** [comment]
> @npeternel I think you run the tests like `UPDATE=true npm test`, see
> 
> https://github.com/mapbox/cloudfriend/blob/37bb0a57570a5ed4449f0ba8b46472ae896282cf/test/shortcuts.test.js#L9
> 
> . We should get that added to the contributing docs.

@vsmart yeah, i ran that and definitely agree it should be documented. but my fixture output isn't complete. i think it's something obvious i'm missing but can't quite figure it out 🤔

**Comment by rclark** [comment]
> but my fixture output isn't complete

@npeternel nothing jumped out at me as wrong with the fixture data. Could you be more specific?

**Comment by npeternel** [comment]
> > but my fixture output isn't complete
> 
> @npeternel nothing jumped out at me as wrong with the fixture data. Could you be more specific?

@rclark i resolved it this morning, and somehow you fixed my CI... 🤷‍♀ does this look good to go then?

**Comment by rclark** [comment]
The PR looks great to me.

About CI... the tests make API calls to AWS to check that the shortcut fixtures are "valid" cloudformation templates. This API has a low rate limit. When pushing to a branch with a PR, travis launches 4 concurrent executions of test suite, and commonly some of them will fail due to rate limiting. Going to the travis UI and restarting builds fixes this. We should come up with a fix to this, worth a ticket.

**Pull request merged**